### ISSUE TITLE
Improve Helping Automation

### ISSUE THREAD

**Issue by StasTserk** [StasTserk/foundry-burningwheel#279]
Now that the extended test mechanic exists, there's less value in the "skip tracking" part of the roll modes.

This should be replaced with a helping roll option instead. With that mode enabled, no roll happens. A test at the exponent gets logged (maybe deferred contingent on success? this would be tricky since it would have to be counted on the fate/call-on/deeds reroll too) and a bonus die or two is registered.
